pkg/util/filehelper/file.go

```diff
@@ -152,3 +152,24 @@ func CopyDir(src string, dst string) (err error) {
 
 	return
 }
+
+// IsDirEmpty checks whether directory is empty or not
+func IsEmptyDir(path string) bool {
+	if !Exists(path) {
+		return true
```

Comment:
The questions is "Is this an empty directory" which is really to questions, is this a directory and if so is it empty. A non existent path cannot be an empty directory.